index.html

```diff
@@ -252,10 +252,10 @@
                             <div class="floe-front-news">
                                 <h3>News</h3>
 
-                                <h4>Inclusive Science Lab</h4>
-                                <p class="floe-date">July 19, 2017</p>
-                                <p>We've been working on designs to make science labs more inclusive, and invited two students to try some things we built. They also helped us make a video!</p>
-                                <p><a href="./news/2017-07-19-inclusive-science-lab.html">Read more</a></p>
+                                <h4>Self-Assessment Learning Toolkit</h4>
+                                <p class="floe-date">August 21, 2017</p>
```

Comment:
Three News items appear in the list on the front page. When a new News item is added, the oldest one of the list gets removed. In this case the Nexus Science Lab should remain (and become 2nd in the list), and the news item from IIDEX 2016 should be removed.